prompt.py

```diff
@@ -757,8 +757,12 @@ def load_smart_contract(self, args):
                     return
 
     def show_mem(self):
-        total = resource.getrusage(resource.RUSAGE_SELF).ru_maxrss
-        totalmb = total / 1000000
+        if sys.platform == 'win32':
+            totalmb = "unknown "  # not supported
```

Comment:
is there no way to get the memory usage on windows?

Reply:
I am not an expert of Python. The help of someone is appreciated!

Reply:
can you try this?
https://stackoverflow.com/a/2017608/3869515

Reply:
@ixje, it's not the same, it indicates the total memory of OS.

Reply:
What exactly should the function `show_mem` return?
If the goal is memory currently used by the process then we can use a cross-platform solution instead of `resource`.
https://stackoverflow.com/a/21632554/5644693

Reply:
oh I missed the specifics. One of these 2 should probably do then:
[still using WMI](https://stackoverflow.com/a/938800/3869515) 
[possible better alternative](https://stackoverflow.com/a/21632554/3869515)

Reply:
Btw, I've just tested it on Ubuntu and it seems like `resource.getrusage(resource.RUSAGE_SELF).ru_maxrss` is the maximum value over all previous `process.memory_info().rss`. So what do we really need: the maximum or the current?

Reply:
We want to show as close as possible the *current* memory consumption.